docs/user-guide/Examples.md

```diff
@@ -48,6 +48,9 @@ When specifying example payloads through an example file, set *DiscoverExamples*
 
 The format of the example file is as follows.  Note that either an external file or inline example can be specified.  Inline examples are useful for initial setup/debugging, but we recommend using external files to maintain examples separately for different requests.
 
+For specifying example body payloads, the special keyword ```__body__``` may be used
```

Comment:
Can you please perhaps add here what the effect of this special keyword is? For instance, does it override/skip schema validation when injecting the example body?

In other words, when should one use this new keyword and why?

Thanks!

Reply:
Yes, thanks, I will make this language more direct (it is recommended to use this field when specifying example bodies).

Reply:
OK. Why is this recommended? (what is the effect?) Thanks!